src/rust/demikernel/libos/mod.rs

```diff
@@ -126,7 +126,22 @@ impl LibOS {
         result
     }
 
-    /// Opens an existing memory queue.
+    /// Creates a new memory queue and connects to producer end.
+    pub fn create_pipe_for_push(&mut self, name: &str) -> Result<QDesc, Fail> {
```

Comment:
Since this is just a temporary solution, shouldn't we annotate it as deprecated?

Reply:
That's true, we could. I wasn't sure if it is weird to add a deprecated function. :)

Reply:
I find it important :) I added that...